src/main/java/org/usrv/http/Server.java

```diff
@@ -72,54 +72,71 @@ public void stop() {
         shouldRun = false;
     }
 
-    private void handleRequest(Socket socket) {
-        String requestId = UUID.randomUUID().toString();
-        MDC.put("requestId", requestId);
-
-        try (
-                socket;
-                BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
-                PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
-        ) {
+private void handleRequest(Socket socket) {
```

Comment:
This block needs to have the same indentation as the rest of the code. Right now, this looks off and creates unnecessary diffs which makes it hard to review. If you update that, I'll have another look!